cmd/gangue/portage.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+)
+
+// Read default parameters from the host for building PORTAGE_BINHOST
+func getReleaseInfo() (board, version string) {
+	contents, err := ioutil.ReadFile("/usr/share/coreos/release")
```

Comment:
If this command is expected to be used in the SDK I'd recommend using VersionsFromManifest and DefaultBoard to be consistent with all other tools.